Grid/Grid.php

```diff
@@ -632,7 +633,7 @@ protected function processMassActions($actionId)
                         $action->getParameters()
                     );
 
-                    $subRequest = $this->container->get('request')->duplicate([], null, $path);
+                    $subRequest = $this->request->duplicate(array(), null, $path);
```

Comment:
@ylastapis yes, you've done right, I was asking to other members it it's the case to uniform every call that use `container`